hivemind/averaging/key_manager.py

```diff
@@ -93,11 +94,11 @@ async def get_averagers(self, group_key: GroupKey, only_active: bool) -> List[Tu
             logger.debug(f"Allreduce group not found: {group_key}, creating new group.")
             return []
         averagers = [
-            (key, entry.expiration_time)
+            (Endpoint.from_base58(key), entry.expiration_time)
             for key, entry in result.value.items()
             if key != self.RESERVED_KEY_FOR_NBITS and (not only_active or entry.value is True)
         ]
-        num_active_averagers = len([key for key, entry in result.value.items() if entry.value is True])
+        num_active_averagers = sum(1 for entry in result.value.values() if entry.value is True)
```

Comment:
Also, `entry` is slightly unclear here (that's why the RHS expression is hard to decipher on its own) — consider renaming